docs/install-and-run.md

```diff
@@ -1,208 +1,239 @@
-     * Download EarlGrey's source code from the [Latest Release](https://github.com/google/EarlGrey/archive/1.0.0.zip)
+   1. Download the source for EarlGrey.
+     1. Download EarlGrey's source code from the
+        [Latest Release](https://github.com/google/EarlGrey/releases/tag/1.8.0).
+
+     2. Unzip and go to the `EarlGrey-x.y.z/EarlGrey` directory
+        that contains **EarlGrey.xcodeproj**.
 
-     * Unzip and go to the `EarlGrey-1.0.0/EarlGrey` directory that contains **EarlGrey.xcodeproj**.
+     3. Open the **EarlGrey.xcodeproj** file.
 
-     * Open the **EarlGrey.xcodeproj** file and build the EarlGrey scheme.
+     4. Build the EarlGrey scheme.
 
-      Your EarlGrey folder structure should now look like this:
+        **Note:** As part of the initial build step, a script [**setup-earlgrey.sh**](https://github.com/google/EarlGrey/tree/master/Scripts/setup-earlgrey.sh)
+        will be run to download all the required dependencies. Without it, you might find dependencies
+        like `fishhook` and `OCHamcrest` shown as missing in the folder structure. Ensure that you are online for this step.
 
-      <img src="images/image03.png" width="200">
+        Your EarlGrey folder structure should now look like this:
 
-      And your EarlGrey Project should look like this:
+        <img src="images/image03.png" width="200">
 
-      <img src="images/image04.png" width="200">
+        And your EarlGrey Project should look like this:
 
-  2. As part of the initial build step, a script **setup-earlgrey.sh** will be run to download all the required dependencies. Without it, you might find dependencies like `fishhook` and `OCHamcrest` shown as missing in the folder structure.
+        <img src="images/image04.png" width="200">
 
 #### Step 2: Add EarlGrey as a dependency of the project which contains your app under test
 
-  1. Close **EarlGrey.xcodeproj** so that it is no longer open in any Xcode window. Once closed, drag **EarlGrey.xcodeproj** from its directory into your App’s project or workspace in Xcode. To verify this, you should find `EarlGrey` in the list of targets of your app in Xcode:
+   1. Close **EarlGrey.xcodeproj** so that it is no longer   
+      open in any Xcode window. Once closed, drag **EarlGrey.xcodeproj** from its directory into your App’s project or workspace in Xcode. To verify this, you should find `EarlGrey` in the list of targets of your app in Xcode:
 
       <img src="images/image05.png" width="350">
 
-  2. Add **EarlGrey.framework** as a dependency of your project’s Test Target:
+   2. Add **EarlGrey.framework** as a dependency of your  
+      project’s Test Target:
 
-    ```
-    Project → Test Target → Build Phases → Link Binary With Libraries → + (Add Sign) → EarlGrey.framework`
-    ```
-
-  3. Add EarlGrey as a Target Dependency to the Test Target:
+      ```
+      **Project** → **Test Target** → **Build Phases** → **Link Binary With Libraries** → **+ (Add Sign)** → **EarlGrey.framework**
```

Comment:
** isn't bolding the texts

plz view in md viewer and fix such issues. I've spotted a lot of them already.